### ISSUE TITLE
Cruster 0.4.5: Could not find certificate file upon first run

### ISSUE THREAD

**Issue by svbrunov** [sinKettu/cruster#6]
Dear Andrey Ivanov (@sinKettu),

First of all, thank you for making and sharing the «Cruster» program!

This GitHub issue represents a defect.

# Scenario
## Preconditions
The first run or, generally speaking, the «working» directory (`~/.cruster`) is empty.

## Steps to reproduce
```
$ rm -rf ~/.cruster
$ mkdir ~/.cruster
$ cruster
```

## Expected result
The program starts successfully (without error messages).

## Actual result
The program fails to start with the error message:

```
$ cruster
Error: ConfigError("Could not find file or dir at relative path 'cruster.cer' neither in workplace nor working dir")
```

# Root cause analysis
The configuration validation implemented by the `config::handle_user_input()` function is unnecessarily restrictive: it requires the existence of the certificate file and the key file.
However, these files would be handled (created, if do not exist) later on by the `utils::generate_key_and_cer()` function.

# Solution
Update the configuration validation not to require the existence of the certificate file and the key file.

## Pull request
[Cruster 0.4.5: Config: Hotfix: Could not find certificate file upon first run by svbrunov · Pull Request #7 · sinKettu/cruster](https://github.com/sinKettu/cruster/pull/7).

Best regards,
Sergey Vyacheslavovich Brunov.

**Comment by sinKettu** [comment]
Hello!
Thank you for you feedback!
You are right, I am sorry for negative experience, but unfortunately I have not enough time to fully cover Cruster with tests, even manual.
I am going to fix it soon.

**Comment by sinKettu** [comment]
As a pretty fast workaround, you can generate certificates with openssl or other similar tool. You only have to set for you certificate flag `IsCa`. 
For example: 
``` shell
openssl req -x509 \
            -sha256 -days 356 \
            -nodes \
            -newkey rsa:2048 \
            -subj "/CN=cruster.local" \
            -keyout .cruster/cruster.key -out .cruster/cruster.crt
```

**Comment by sinKettu** [comment]
Fixed with PR #7 , Thanks to @svbrunov .
Probably later I'll push hotfix into master without awaiting for `0.4.5` version.
